other/js/websockify.js

```diff
@@ -26,9 +26,9 @@ var argv = require('optimist').argv,
 
 
 // Handle new WebSocket client
-new_client = function(client) {
+new_client = function(client, req) {
     var clientAddr = client._socket.remoteAddress, log;
-    console.log(client.upgradeReq.url);
+    console.log(req.url);
```

Comment:
It is probably better to use something like this:

```js
console.log(req ?  req.url : client.upgradeReq.url);
```

so this will work with all `ws` versions.